gcloud/logging/config_service_v2_api.py

```diff
@@ -0,0 +1,208 @@
+# Copyright 2015 Google Inc. All rights reserved.
```

Comment:
This copyright is 2015 and the one in `gcloud/logging/__init__.py` is 2016. Which do you want?